docs/reference/release-notes/8.0.0-alpha1.asciidoc

```diff
@@ -1,44 +1,520 @@
+* [Rest Api Compatibility] Typed indexing stats {es-pull}74181[#74181] (issues: {es-issue}47203[#47203], {es-issue}51816[#51816], {es-issue}54160[#54160])
+* [Rest Api Compatibility] Types for Percolate Query Api {es-pull}74698[#74698] (issues: {es-issue}46985[#46985], {es-issue}51816[#51816], {es-issue}54160[#54160], {es-issue}74689[#74689])
+* [Rest Api Compatibility] Validate Query typed api {es-pull}74171[#74171] (issues: {es-issue}46927[#46927], {es-issue}51816[#51816], {es-issue}54160[#54160])
+* [Rest Api Compatibility] Voting config exclusion exception message {es-pull}75406[#75406] (issues: {es-issue}51816[#51816], {es-issue}55291[#55291])
+* [Rest Api Compatibility] `MoreLikeThisQuery` with types {es-pull}75123[#75123] (issues: {es-issue}42198[#42198], {es-issue}51816[#51816], {es-issue}54160[#54160])
+* [Rest Compatible Api] update and delete by query using size field {es-pull}69606[#69606]
+
+Infra/Scripting::
+* Update `DeprecationMap` to `DynamicMap` {es-pull}56149[#56149] (issue: {es-issue}52103[#52103])
+
+Infra/Settings::
+* Fixed inconsistent Setting.exist() {es-pull}46603[#46603] (issue: {es-issue}41830[#41830])
+* Remove setting `index.optimize_auto_generated_id` (#27583) {es-pull}27600[#27600] (issue: {es-issue}27583[#27583])
+
+License::
+* Add deprecated `accept_enterprise` param to /_xpack {es-pull}58220[#58220] (issue: {es-issue}58217[#58217])
+* Support "accept_enterprise" param in get license {es-pull}50067[#50067] (issue: {es-issue}49474[#49474])
+
+MULTIPLE AREA LABELS::
+* Add recovery state tracking for Searchable Snapshots {es-pull}60505[#60505]
+* Configure security for the initial node cli {es-pull}74868[#74868]
+* QL: Improve removal of items during iteration {es-pull}71193[#71193]
+* QL: Make canonical form take into account children {es-pull}71266[#71266]
+* QL: Polish optimizer expression rule declaration {es-pull}71396[#71396]
+* QL: Propagate nullability constraints across conjunctions {es-pull}71187[#71187] (issue: {es-issue}70683[#70683])
+* [Rest Api Compatibility] Allow to use size -1 {es-pull}75342[#75342] (issues: {es-issue}51816[#51816], {es-issue}69548[#69548], {es-issue}70209[#70209])
+* [Rest Api Compatibility] Ignore `use_field_mapping` option for docvalue {es-pull}74435[#74435] (issue: {es-issue}55622[#55622])
+* [Rest Api Compatibility] `_time` and `_term` sort orders {es-pull}74919[#74919] (issues: {es-issue}39450[#39450], {es-issue}51816[#51816])
+
+Machine Learning::
+* Change version in serialization code in `TimingStats.java` to 7.4.0 {es-pull}44905[#44905] (issue: {es-issue}29857[#29857])
+* Limit the input sequence length for NLP processors {es-pull}74905[#74905]
+* Merge the pytorch-inference feature branch {es-pull}73660[#73660] (issues: {es-issue}70713[#70713], {es-issue}71035[#71035], {es-issue}71177[#71177], {es-issue}71323[#71323], {es-issue}71679[#71679], {es-issue}72218[#72218], {es-issue}73523[#73523])
```

Comment:
```suggestion
```

I suggest removing this low level detail.  The functionality is better described by "Add a new application for evaluating PyTorch models..." a few lines below.  All this is part of the same change.